src/App.tsx

```diff
@@ -404,6 +405,26 @@ const MainStack = () => {
   );
 };
 
+const linking = {
+  prefixes: [
+    DEEP_LINK_PREFIX,
+    DEEP_LINK_DOMAIN
```

Comment:
I see we've just merged a lint tidy up, not yet added Husky etc - can we run `npm run lint:fix` for these.